src/SourceGenerators/Uno.UI.Tasks/Content/Uno.UI.Tasks.targets

```diff
@@ -33,9 +33,9 @@
 					  ItemName="GeneratedFiles" />
 	</ResourcesGenerationTask_v0>
 	<ItemGroup>
-	  <BundleResource Condition="'$(XamarinProjectType)'=='ios'" Include="@(GeneratedFiles)" />
-	  <AndroidResource Condition="'$(XamarinProjectType)'=='android'" Include="@(GeneratedFiles)" />
-	  <EmbeddedResource Condition="'$(XamarinProjectType)'=='wasm' or '$(XamarinProjectType)'==''" Include="@(GeneratedFiles)" />
+	  <BundleResource Condition="'%(GeneratedFiles.UnoResourceTarget)' =='iOS'" Include="@(GeneratedFiles)" />
+	  <AndroidResource Condition="'%(GeneratedFiles.UnoResourceTarget)' =='Android'" Include="@(GeneratedFiles)" />
+	  <EmbeddedResource Condition="'%(GeneratedFiles.UnoResourceTarget)' =='Uno'" Include="@(GeneratedFiles)" />
```

Comment:
Uno ? as a target?

Reply:
It's a target because the reader of the generated resource file is Uno itself, and it's cross platform.